tracker/tests.py

```diff
@@ -1,3 +1,65 @@
-from django.test import TestCase
-
+from django.test import TestCase, Client
+from django.urls import reverse, resolve
+import json
+from .models import *
+from .views import *
+import datetime
 # Create your tests here.
+
+class TestModels(TestCase):
+	"""docstring for ClassName"""
+	def setUp(self):
+		Task.objects.create(
+			name = 'xx',
+			planning = '07:04',
+			estimate = '02:50',
+			realtime = '04:52',
+			risk = 'OK',
+			level = 'easy',
+			notes = 'azerty',
+			estimateb_by_calc = '4:27',
+			correctness = 50,
+		)
+
+	def test_Task(self):
+		self.new = Task.objects.get(risk = 'OK',)
+		self.assertEquals(self.new.realtime, datetime.time(4, 52))
+
+
+class TestUrls(TestCase):
+	"""docstring for ClassName"""
+	def test_urls(self):
+		url = reverse('home')
+		self.assertEquals(resolve(url).func.view_class, TaskView)
+
+
+class TestView(TestCase):
+	"""docstring for ClassName"""
+	def setUp(self):
+		Task.objects.create(
+			name = 'xxx',
+			planning = '09:04',
+			estimate = '07:50',
+			realtime = '02:52',
+			risk = 'OK',
+			level = 'easy',
+			notes = 'cool',
+			estimateb_by_calc = '6:17',
+			correctness = 50,
+		)
+		self.client = Client()
+		self.url = reverse('home')
+
+
+
+	def test_views_POST(self):
+		self.post = Task.objects.get(name='xxx')
```

Comment:
There is a difference between the name of your test and what it is currently testing (If object has been created).
To check if the object has been created, you could just check for the Task.object.count(). If equals 1 => created else not created.